jenkins-slaves/jenkins-slave-graalvm/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM quay.io/openshift/origin-jenkins-agent-maven:4.2
+ARG GRAALVM_VERSION=19.3.1
+
+ADD settings.xml $HOME/.m2/settings.xml
+
+
+USER root
+RUN yum install -y gcc glibc-devel zlib-devel libstdc++-static
```

Comment:
```
STEP 5: RUN yum install -y gcc glibc-devel zlib-devel libstdc++-static
Loaded plugins: ovl, product-id, search-disabled-repos, subscription-manager
This system is not registered with an entitlement server. You can use subscription-manager to register.
http://base-4-2.ocp.svc/rhel-fast-datapath/repodata/repomd.xml: [Errno 14] curl#6 - "Could not resolve host: base-4-2.ocp.svc; Unknown error"
Trying other mirror.


 One of the configured repositories failed (rhel-fast-datapath),
 and yum doesn't have enough cached data to continue. At this point the only
 safe thing yum can do is fail. There are a few ways to work "fix" this:

     1. Contact the upstream for the repository and get them to fix the problem.

     2. Reconfigure the baseurl/etc. for the repository, to point to a working
        upstream. This is most often useful if you are using a newer
        distribution release than is supported by the repository (and the
        packages for the previous distribution release still work).

     3. Run the command with the repository temporarily disabled
            yum --disablerepo=rhel-fast-datapath ...

     4. Disable the repository permanently, so yum won't use it by default. Yum
        will then just ignore the repository until you permanently enable it
        again or use --enablerepo for temporary usage:

            yum-config-manager --disable rhel-fast-datapath
        or
            subscription-manager repos --disable=rhel-fast-datapath

     5. Configure the failing repository to be skipped, if it is unavailable.
        Note that yum will try to contact the repo. when it runs most commands,
        so will have to try and fail each time (and thus. yum will be be much
        slower). If it is a very temporary problem though, this is often a nice
        compromise:

            yum-config-manager --save --setopt=rhel-fast-datapath.skip_if_unavailable=true

failure: repodata/repomd.xml from rhel-fast-datapath: [Errno 256] No more mirrors to try.
http://base-4-2.ocp.svc/rhel-fast-datapath/repodata/repomd.xml: [Errno 14] curl#6 - "Could not resolve host: base-4-2.ocp.svc; Unknown error"
error building at STEP "RUN yum install -y gcc glibc-devel zlib-devel libstdc++-static": error while running runtime: exit status 1
ERRO exit status 1
```

Reply:
Not sure how to fix that... The underlying cluster SHOULD have access to that YUM repo

Reply:
IIRC the `quay.io/openshift/origin-jenkins-agent-maven` image is based on UBI which can't/shouldn't be using RHEL repos.